create.sh

```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
-
-helm install --name data --namespace tick ./influxdb/
-helm install --name polling --namespace tick ./telegraf-s/
-helm install --name hosts --namespace tick ./telegraf-ds/
-helm install --name alerts --namespace tick ./kapacitor/
-helm install --name dash --namespace tick ./chronograf/
-kubectl get svc -w --namespace tick -l app=dash-chronograf
```

Comment:
Why do we need to completely delete this file? Can we change it instead to accept a parameter for eks (or other environments) so that the user is still able to spin everything up with a single command?